### PR TITLE
Blacken the source code and sort imports

### PR DESCRIPTION
**Pull request by samj1912**
Thanks for submitting a PR!

Please make sure to check for the following items:
- [ ] Add unit tests and integration tests where applicable.  
      If you've added an error code or changed an error code behavior,
      you should probably add or change a test case file under `tests/test_cases/` and add 
      it to the list under `tests/test_definitions.py`.  
      If you've added or changed a command line option,
      you should probably add or change a test in `tests/test_integration.py`.
- [ ] Add a line to the release notes (docs/release_notes.rst) under "Current Development Version".  
      Make sure to include the PR number after you open and get one.
   
Please don't get discouraged as it may take a while to get a review.

### CODE REVIEW

File: src/pydocstyle/checker.py [hunk 1]
```diff
@@ -91,42 +108,55 @@ class ConventionChecker:
     # " random         : test" where random will be captured as the param
     # "  random_t (Test) : test  " where random_t will be captured as the param
     GOOGLE_ARGS_REGEX = re(
-                        # Matches anything that fulfills all the following conditions:
```
Comment (Nurdok) [inline src/pydocstyle/checker.py:94]: Can you move each comment to the line above the regex it describes? This hurts readability IMO.

**Review by Nurdok — CHANGES_REQUESTED** [review]

File: src/pydocstyle/checker.py (hunk 1 above)
Comment (samj1912) [reply src/pydocstyle/checker.py:94]: Fixed :)

**Comment by samj1912** [comment]
@Nurdok bump

**Review by Nurdok — APPROVED** [review]

**Comment by Nurdok** [comment]
Sorry for the delay @samj1912. Between the first review and now, my wife managed to give birth to a healthy baby boy :)

**Pull request merged**